config.sample.yaml

```diff
@@ -74,6 +74,12 @@ services:
     Config:
       api_key: "2356saaqfhgfe"
 
+  - ID: "google_service"
+    Type: "google"
+    UserID: "@goneb:localhost" # requires a Syncing client
+    Config:
+      api_key: "AIzaSyA4FD39m9pN-hiYf2NRU9x9cOv5tekRDvM"
```

Comment:
Key changed. Fixed in https://github.com/matrix-org/go-neb/commit/19845fcc2569652787fa26250f597f41f9c19988.